decidim/decidim
# Migrate Survey export to use HexaPDF

#### :tophat: What? Why?
This PR aims to change the syntax of the survey PDF export to use the hexaPDF library, so that we can deprecate and remove wicked_pdf. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #13553
- Related to #12135

#### Testing
1. Create a survey component 
2. Add some answers using multiple users
3. You should receive a copy of your answers as PDF 
4. Visit the survey admin panel, and choose export as PDF 
5. Visit the survey answers page, and choose individual export of answers
6. Compare the results with what exists in the nightly ( or what is attached to this PR )

[wkhtmltopdf-survey.pdf](https://github.com/user-attachments/files/17668711/wkhtmltopdf-survey.pdf)
[hexapdf-survey.pdf](https://github.com/user-attachments/files/17670869/hexapdf-survey.pdf)


### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
